src/background/stores/card-selection-store.ts

```diff
@@ -71,14 +94,18 @@ export class CardSelectionStore extends BaseStoreImpl<CardSelectionStoreData> {
         if (
             !payload ||
             !this.state.rules[payload.ruleId] ||
-            this.state.rules[payload.ruleId].cards[payload.resultInstanceUid] === undefined
+            this.state.rules[payload.ruleId].cards[
+                payload.resultInstanceUid
+            ] === undefined
         ) {
             return;
         }
 
         const rule = this.state.rules[payload.ruleId];
 
-        rule.cards[payload.resultInstanceUid] = !rule.cards[payload.resultInstanceUid];
+        rule.cards[payload.resultInstanceUid] = !rule.cards[
```

Comment:
I get the reasons for the shorter line lengths, but it feels like we have some cases where we sacrifice the ease of reading the code. This line is just one example--the old code is obvious that we're flipping the state of `rule.cards[payload.resultInstanceUid]`. In the new code I need to "re-stitch" across 3 lines.

Reply:
All of these are part of automated prettier `format:fix` run. So, I didn't control this except for changing line width to 80 in the prettier config. But that being said, I feel we should discuss this but I am not unsure if we want to block this PR or not.